tests/test_spd_matrices.py

```diff
@@ -33,19 +33,18 @@ def setUp(self):
     def test_random_uniform_and_belongs(self):
         point = self.space.random_uniform()
         result = self.space.belongs(point)
-        expected = gs.array([[True]])
+        expected = gs.array(True)
         self.assertAllClose(result, expected)
 
     @geomstats.tests.np_and_tf_only
     def test_random_uniform_and_belongs_vectorization(self):
         """
-        Test that the random uniform method samples
-        on the hypersphere space.
+        same: fix belongs.
```

Comment:
Use `# FIXME` syntax, it will be easier to `grep` when we have "Fix it days"

Reply:
right! and belongs is already fixed...